up.sh

```diff
@@ -39,5 +39,36 @@ sed -i "s/replace_with_ip/$NFS_SRV_IP/g"  ./netboot/menu.ipxe
 wget https://cloud.3mdeb.com/index.php/s/UQQVYrNIhg7ddwj/download -O kernels.tar.gz
 tar -xzvf kernels.tar.gz -C ./netboot && rm kernels.tar.gz
 
+# download root file systems
+wget https://cloud.3mdeb.com/index.php/s/9b8h6WmJcNsuB57/download -O debian-stable.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/fzQ2FaRTdMvzXqO/download -O xen.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/AQuUdsYkBzO9UJz/download -O core.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/rUZPwRHOjxpSxN4/download -O voyage-0.11.0_amd64.tar.gz
+
+NFS_EXPORT_DIR="./nfs-export"
+mkdir -p $NFS_EXPORT_DIR
+
+# extract debian
+DEBIAN_DIR="$NFS_EXPORT_DIR/debian"
+tar -xvpzf debian-stable.tar.gz -C $NFS_EXPORT_DIR --numeric-owner
+mv $NFS_EXPORT_DIR/debian-stable $DEBIAN_DIR
+
+# extract xen
+XEN_DIR="$NFS_DIR/xen"
+tar -xvpzf xen.tar.gz -C $NFS_EXPORT_DIR --numeric-owner
+
+# extract voyage
+VOYAGE_DIR="$NFS_EXPORT_DIR/voyage"
+tar -xzvf voyage-0.11.0_amd64.tar.gz -C $VOYAGE_DIR
+
+# extract core ??
```

Comment:
This is correct. Core runs only in memory so it requires kernel and initrd to be fetched via http in iPXE